### PR TITLE
Migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by DmitryTsepelev**

### CODE REVIEW

File: .github/workflows/bench.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        gemfile: ["gemfiles/rails_6_0.gemfile"]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/cache@v2
+      with:
+        path: /home/runner/bundle
+        key: bundle-${{ matrix.ruby }}-${{ matrix.gemfile }}-${{ hashFiles(matrix.gemfile) }}-${{ hashFiles('**/*.gemspec') }}
+        restore-keys: |
+          bundle-${{ matrix.ruby }}-${{ matrix.gemfile }}-
+    - uses: ruby/setup-ruby@v1
```
Comment (PikachuEXE) [inline .github/workflows/bench.yml:28]: If you use `ruby/setup-ruby`
You can use option `bundler-cache`
and remove `actions/cache` & `bundle install`

File: .github/workflows/bench.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      with:
+        ruby-version: ${{ matrix.ruby }}
+    - name: Install system deps
+      run: |
+        sudo apt-get update
+        sudo apt-get install libsqlite3-dev
+    - name: Bundle install
+      run: |
+        bundle config path /home/runner/bundle
+        bundle config --global gemfile ${{ matrix.gemfile }}
```
Comment (PikachuEXE) [inline .github/workflows/bench.yml:38]: To use `bundler-cache` option this should be moved to `env` as `BUNDLE_GEMFILE` like
https://github.com/PikachuEXE/where_lower/blob/master/.github/workflows/tests.yaml#L41-L42

File: .github/workflows/bench.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        restore-keys: |
+          bundle-${{ matrix.ruby }}-${{ matrix.gemfile }}-
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+    - name: Install system deps
+      run: |
+        sudo apt-get update
+        sudo apt-get install libsqlite3-dev
+    - name: Bundle install
```
Comment (PikachuEXE) [inline .github/workflows/bench.yml:35]: This step can be completely removed when using `bundler-cache`

File: .github/workflows/bench.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.7]
+        gemfile: ["gemfiles/rails_6_0.gemfile"]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/cache@v2
```
Comment (PikachuEXE) [inline .github/workflows/bench.yml:22]: This step can be completely removed when using `bundler-cache`

**Review by PikachuEXE — CHANGES_REQUESTED** [review]
Commented on one file but applies to all 3 files

**Comment by DmitryTsepelev** [comment]
Done, thanks! Also, I've realized that Coveralls was broken, now it's back

**Review by PikachuEXE — APPROVED** [review]

**Pull request merged**